source/clear-linux/tutorials/telemetry-backend/Telemetry-backend.rst

```diff
@@ -0,0 +1,495 @@
+
+#. Choose to install |CL|
+#. Join the :guilabel:`Stability Enhancement Program` to install and
+   enable the telemetrics components
+#. Select the manual installation method with the following settings:
+
+   * Set the hostname to :guilabel:`clr-telem-server`,
+   * Create an administrative user named :guilabel:`clear` and add this user
+     to sudoers,
+   * Add all additional software bundles
+
+Optionally, if you already have a |CL| system set up that you want to install
+the telemetry backend server components to, or to a virtual |CL| machine, you
+need to have a user account that has sudo privileges and the following
+software bundles already installed:
+
+* dev-utils - has the :command:`git` command
+* telemetrics - has the required telemetry applications, see the 
+  :ref:`telemetry-enable` guide for installing and enabling telemetrics.
+
+To determine the software bundles already installed, enter the
+:command:`sudo swupd bundle-list` command to list the current bundles
+installed and then run :command:`sudo swupd bundle-add dev-utils telemetrics`
+if either one is not installed.
+
+Download the clearlinux/telemetrics-backend git repository
+==========================================================
+
+With all prerequisite software bundles installed and logged in with your
+administrative user, from your $HOME directory, run :command:`git` to clone
+the :guilabel:`telemetrics-backend` repository into the
+:file:`$HOME/telemetrics-backend` directory:
+
+.. code-block:: console
+
+   git clone https://github.com/clearlinux/telemetrics-backend
+
+.. note::
+   You may need to set up the :guilabel:`https_proxy` environment variable
+   if you have issues reaching github.com.
+
+Run the deploy.sh script to install the backend server
+======================================================
+
+Change your current working directory to :file:`telemetrics-backend/scripts`
+and then run the :command:`./deploy.sh -h` to see the list of options for the
+:command:`deploy.sh` script:
+
+.. code-block:: console
+
+   cd telemetrics-backend/scripts
+   ./deploy.sh -h
+  
+   Deploy snapshot of the telemetrics-backend
+
+     -a    Perform specified action (deploy, install, migrate, resetdb,
+           restart, uninstall; default: deploy)
+     -d    Distro to deploy to (ubuntu, centos or clr; default: ubuntu)
+     -h    Print these options
+     -H    Set domain for deployment (only accepted value is "localhost" for
+           now)
+     -r    Set repo location to deploy from
+           (default: https://github.com/clearlinux/telemetrics-backend)
+     -s    Set source location (default: "master" branch from git repo)
+     -t    Set source type (tarball, or git; default: git)
+     -u    Perform complete uninstallation
+
+The :command:`deploy.sh` is a bash shell script that allows you to perform the
+following actions:
+
+* deploy - install a complete instance of the telemetrics backend
```

Comment:
Since these are all options after the deploy.sh "command" use the :option: role:
In this line it would be :option:\`deploy\`

Reply:
done